packages/backend/src/models/dynamodb/table-row.ts

```diff
@@ -138,3 +138,20 @@ export const deleteTableRow = async ({
     wrapDynamoDBError(e)
   }
 }
+
+export const getAllTableRows = async ({
```

Comment:
same nit with naming. Would prefer we have something like `getAllTableRows_DANGEROUS`

Reply:
as discussed, there wont be a need for pagination since we are doing row virtualization